znoyder/templater.py

```diff
@@ -58,6 +60,20 @@ def generate_zuul_config(path: str, name: str,
 
     jobs_dict = defaultdict(list)
 
+
+# Making sure the template provided is available
```

Comment:
Just a late hint: this comment is not necessary, with `git blame` we can find all the details.